doc/manipulating_images/data_preparation.rst

```diff
@@ -453,16 +445,16 @@ A single seed is a sphere defined by the radius (in millimeters) and the
 coordinates (typically MNI or TAL) of its center.
 
 Using :class:`NiftiSpheresMasker` needs to define a list of coordinates.
-"seeds" argument takes a list of 3D coordinates (tuples) of the spheres centers,
+`seeds` argument takes a list of 3D coordinates (tuples) of the spheres centers,
 they should be in the same space as the images.
-Seeds can overlap spatially and are represented in a binary present-nonpresent
+Seeds can overlap spatially and are represented in a binary present/nonpresent
 coding (no weighting).
 Below is an example of a coordinates list of four seeds from the default mode network::
 
   >>> dmn_coords = [(0, -52, 18), (-46, -68, 32), (46, -68, 32), (0, 50, -5)]
 
-"radius" is an optional argument that takes a real value in millimeters.
-If no value is given for the "radius" argument, the single voxel at the given
+`radius` is an optional argument that takes a real value in millimeters.
+If no value is given for the `radius` argument, the single voxel at the given
```

Comment:
perhaps less ambiguous: "the radius will only contain the single voxel at the radius center"